src/Numerics/DGMethods/DGModel.jl

```diff
@@ -655,7 +654,7 @@ function reverse_indefinite_stack_integral!(
     dim = dimensionality(grid)
     N = polynomialorders(grid)
     Nq = N .+ 1
-    Nqk = dim == 2 ? 1 : Nq[dim]
+    Nqk = dim == 2 ? 1 : Nq[2]
```

Comment:
Could we change this name to `Nqj` or `Nq2` ? `Nqk` is defined differently in other places.

Reply:
Done in [4ad0db5](https://github.com/CliMA/ClimateMachine.jl/pull/1833/commits/4ad0db58b11701d82f4ed967f2dbda429ab4c9c6)